hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/CopyFromLocalOperation.java

```diff
@@ -356,13 +356,9 @@ private void checkDestination(
   private Path getFinalPath(Path src) throws PathIOException {
     URI currentSrcUri = src.toUri();
     URI relativeSrcUri = source.toUri().relativize(currentSrcUri);
-    if (relativeSrcUri.equals(currentSrcUri)) {
-      throw new PathIOException("Cannot get relative path for URI:"
```

Comment:
is it possible that`currentSrcUri` and `relativeSrcUri` are both empty?

Reply:
Interesting question. My expectation is no, but I haven't checked. I'll take a peek.

Reply:
Closing the loop on this, because this uses `Path`, it can't be empty: https://github.com/HubSpot/hadoop/blob/9029bba5dcb5a0379b27b32d811756de965d706c/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/Path.java#L165-L175